tests/utils_tests/test_autoreload.py

```diff
@@ -124,6 +125,11 @@ def test_weakref_in_sys_module(self):
         self.addCleanup(lambda: sys.modules.pop('time_proxy', None))
         list(autoreload.iter_all_python_module_files())  # No crash.
 
+    def test_module_without_spec(self):
+        module = types.ModuleType("test_module")
+        del module.__spec__
+        autoreload.iter_modules_and_files((module,), frozenset([]))
```

Comment:
You can pass `set()`, no need to create a list to initialize an empty `frozenset` (or `set`), it be great to assert the return value as well.